javascript/fluent.js

```diff
@@ -128,13 +128,18 @@
 			}
 		});
 
-		$('div.LocalisedField input.LocalisedField').entwine({
+		$('div.LocalisedField .LocalisedField').entwine({
 			/**
 			 * Check for changes against the default value
 			 */
 			onchange: function() {
-				var newValue = this.val();
-				var defaultValue = $.parseJSON(this.data('default-locale-value'));
+				if (this.is('input')) {
+					var newValue = this.val();
+				} else if (this.is('textarea')) {
+					var newValue = this.text();
```

Comment:
Damn - I'd become pretty happy having gotten rid of the damn WYSIWYG editor from my environment :) Anywho - it does update the textarea, but the iframe that is run by Tiny doesn't update with the value in the hidden textarea. Pushing an update for Tiny now...